Meta/run.py

```diff
@@ -639,13 +665,8 @@ def set_up_boot_drive(config: Configuration):
         config.add_devices(["ahci,id=boot-drive-ahci", "ide-hd,drive=boot-drive,bus=boot-drive-ahci.0"])
         config.kernel_cmdline.append("root=ahci0:0:0")
     if config.boot_drive_type == BootDriveType.NVMe:
-        config.add_devices(
-            [
-                "i82801b11-bridge,id=bridge4",
-                "nvme,serial=deadbeef,drive=boot-drive,bus=bridge4,logical_block_size=4096,physical_block_size=4096",
-            ]
-        )
-        config.kernel_cmdline.append("root=nvme0:1:0")
+        config.add_devices(["nvme,serial=deadbeef,drive=boot-drive"])
```

Comment:
I would prefer keeping this bridge on x86, like in your first version of this PR.
Otherwise LGTM

Reply:
Whoops. re-added.